src/Directory.Packages.props

```diff
@@ -1,57 +1,66 @@
+        <PackageVersion Include="CommandLineParser" Version="2.9.1" />
+        <PackageVersion Include="coverlet.msbuild" Version="6.0.2" />
+        <PackageVersion Include="coverlet.collector" Version="6.0.2" />
+        <PackageVersion Include="HotChocolate" Version="12.22.0" />
+        <PackageVersion Include="HotChocolate.AspNetCore" Version="12.22.0" />
+        <PackageVersion Include="HotChocolate.AspNetCore.Authorization" Version="12.22.0" />
+        <PackageVersion Include="HotChocolate.Types.NodaTime" Version="12.22.0" />
+        <PackageVersion Include="Humanizer" Version="2.14.1" />
+        <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
+        <PackageVersion Include="DotNetEnv" Version="3.0.0" />
+        <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.22.0" />
+        <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
+        <PackageVersion Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
+        <PackageVersion Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+        <PackageVersion Include="Microsoft.Azure.Cosmos" Version="3.38.1" />
+        <!--When updating Microsoft.Data.SqlClient, update license URL in scripts/notice-generation.ps1-->
+        <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.0" />
+        <PackageVersion Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.1" />
+        <PackageVersion Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
+        <PackageVersion Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.22.0" />
+        <PackageVersion Include="Microsoft.IdentityModel.JsonWebTokens" Version="7.5.0" />
+        <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+        <PackageVersion Include="Microsoft.OData.Edm" Version="7.20.0" />
+        <PackageVersion Include="Microsoft.OData.Core" Version="7.20.0" />
+        <PackageVersion Include="Microsoft.OpenApi" Version="1.6.14" />
+        <PackageVersion Include="Microsoft.OpenApi.Readers" Version="1.6.14" />
+        <PackageVersion Include="Moq" Version="4.18.2" />
+        <PackageVersion Include="MSTest.TestAdapter" Version="3.2.2" />
+        <PackageVersion Include="MSTest.TestFramework" Version="3.2.2" />
+        <PackageVersion Include="MySqlConnector" Version="2.1.5" />
+        <PackageVersion Include="Newtonsoft.Json" Version="13.0.2" />
+        <PackageVersion Include="Npgsql" Version="7.0.1" />
+        <PackageVersion Include="Polly" Version="7.2.3" />
+        <PackageVersion Include="NJsonSchema" Version="10.9.0" />
+        <PackageVersion Include="StyleCop.Analyzers" Version="1.1.118" />
+        <PackageVersion Include="Swashbuckle.AspNetCore.SwaggerUI" Version="6.5.0" />
+        <PackageVersion Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
+        <PackageVersion Include="System.Drawing.Common" Version="8.0.3" />
```

Comment:
Does our NOTICE file mention the updated version? We might have to test the official pipeline with this PR to find that info

Reply:
updated the notice file creation script and will check output.

Reply:
noticefile generates successfully with newest sqlclient version specified.